app/controllers/view_components_controller.rb

```diff
@@ -4,4 +4,10 @@
 
 class ViewComponentsController < Rails::ApplicationController # :nodoc:
   include ViewComponent::PreviewActions
```

Comment:
Looking at this now, I think there was a reason we moved all of the actions to a module. Mind putting this new action in `PreviewActions`?